Funcky.Test/Extensions/EnumerableExtensions/ConcatStringsTest.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Funcky.Extensions;
+using Xunit;
+
+namespace Funcky.Test.Extensions.EnumerableExtensions
+{
+    public class ConcatStringsTest
+    {
+        [Fact]
+        public void ConcatenatingAnEmptySetOfStringsReturnsAnEmptyString()
+        {
+            var empty = Enumerable.Empty<string>();
+
+            Assert.Equal(string.Empty, empty.ConcatToString());
+        }
+
+        [Fact]
+        public void ConcatenatingASetWithExactlyOneElementReturnsTheElement()
+        {
+            var singleElement = Sequence.Return("Alpha");
+
+            Assert.Equal("Alpha", singleElement.ConcatToString());
+        }
+
+        [Fact]
+        public void ConcatenatingAListOfStringsXXXX()
```

Comment:
This seems to be a placeholder name :)

Reply:
Renamed the test accordingly: 93b26d313b9a589626312a7bfecce4a801d4d63c